lua/wikis/commons/Icon/Data.lua

```diff
@@ -116,6 +116,10 @@ return {
 	-- Usage: Dota2 Main Page
 	hub = 'fas fa-external-link-alt',
 
+	-- Usage: Rainbow6 Infobox operator
```

Comment:
```suggestion
	-- Usage: Indicate boolean info in Infobox
```
Usage should be instructions when to use it rather, rather than where it is used (not all existing ones are good)